FtpServer/FTP_Server.sln

```diff
@@ -0,0 +1,28 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.24720.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "FTP_Server", "FTP_Server\FTP_Server.vcxproj", "{E14E9367-264D-4569-966E-2899C66B772B}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{E14E9367-264D-4569-966E-2899C66B772B}.Debug|x64.ActiveCfg = Debug|x64
+		{E14E9367-264D-4569-966E-2899C66B772B}.Debug|x64.Build.0 = Debug|x64
+		{E14E9367-264D-4569-966E-2899C66B772B}.Debug|x86.ActiveCfg = Debug|Win32
+		{E14E9367-264D-4569-966E-2899C66B772B}.Debug|x86.Build.0 = Debug|Win32
+		{E14E9367-264D-4569-966E-2899C66B772B}.Release|x64.ActiveCfg = Release|x64
+		{E14E9367-264D-4569-966E-2899C66B772B}.Release|x64.Build.0 = Release|x64
+		{E14E9367-264D-4569-966E-2899C66B772B}.Release|x86.ActiveCfg = Release|Win32
+		{E14E9367-264D-4569-966E-2899C66B772B}.Release|x86.Build.0 = Release|Win32
```

Comment:
no, you should only commit project file and sources. Remove please debug dir and .sln file.